### PR TITLE
Publish samples

### PR DESCRIPTION
**Pull request by neerajrj**
### Context

mantis-publish-sample : Shows instrumenting a simple java app with the mantis-publish library to push events to mantis.

mantis-publish-web-sample: Shows instrumenting a web application with a servlet filter to push captured request / response information on a per request basis to mantis

### Checklist

- [ x] `./gradlew build` compiles code correctly
- [ x] Added new tests where applicable
- [ x] `./gradlew test` passes all tests
- [ x] Extended README or added javadocs where applicable
- [ x] Added copyright headers for new files from `CONTRIBUTING.md`

### CODE REVIEW

File: mantis-publish-sample/buildDockerImage.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# build the mantis-server-worker fat jar
+../gradlew clean build shadowDistZip
+unzip build/distributions/mantis-examples-mantis-publish-sample-shadow-0.1.0-SNAPSHOT.zip -d build/distributions 
+# build the Docker image that packages the mantis-server-worker along with a running mesos-slave
+docker build -t dev/mantispublish .
```
Comment (jeffchao) [inline mantis-publish-sample/buildDockerImage.sh:7]: Do we intend to publish this to dockerhub? should put as `netflixoss/mantispublish` if so.

File: mantis-publish-web-sample/buildDockerImage.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# build the mantis-server-worker fat jar
+../gradlew clean build
+
+# build the Docker image that packages the mantis-server-worker along with a running mesos-slave
+docker build -t dev/mantispublishweb .
```
Comment (jeffchao) [inline mantis-publish-web-sample/buildDockerImage.sh:7]: Same for this one.

**Review by jeffchao — APPROVED** [review]

File: mantis-publish-web-sample/buildDockerImage.sh (hunk 2 above)
Comment (neerajrj) [reply mantis-publish-web-sample/buildDockerImage.sh:7]: Yes plan to do that in the next iteration. Wanted to get the sample out first.

File: mantis-publish-sample/buildDockerImage.sh (hunk 1 above)
Comment (neerajrj) [reply mantis-publish-sample/buildDockerImage.sh:7]: Yes plan to do that in the next iteration. Wanted to get the sample out first.

**Pull request merged**